spring-integration-jdbc/src/main/java/org/springframework/integration/jdbc/channel/PostgresChannelMessageTableSubscriber.java

```diff
@@ -297,7 +297,7 @@ public boolean isRunning() {
 		return this.latch.getCount() > 0;
 	}
 
-	private static String getKey(Object input) {
+	private static @Nullable String getKey(@Nullable Object input) {
```

Comment:
I believe same approach for the `Objects.requireNonNull()` as for similar places in other classes we have discussed here.